docs/connecting-to-a-lit-network/lit-blockchains/migrations/migrating-to-vesuvius.md

```diff
@@ -0,0 +1,60 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Migrating From Chronicle to Chronicle Vesuvius
```

Comment:
imo we should title this 'migrating to the Lit network datil' (make it about the network, not the chain)

Reply:
I'm on the fence for this one. I understand your point, but it is the chain that's changed and causing PKP, encrypted data, etc to be migrated. We'd also be linking users to the Datil network sections of either the `Testnets` or `Mainnets` page and those section will have the blurb:

> If you are currently on one of the Lit networks that utilize the Chronicle blockchain (i.e. Cayenne, Manzano, and/or Habanero), please refer to [this guide](http://localhost:3000/connecting-to-a-lit-network/lit-blockchains/migrations/migrating-to-vesuvius) to learn how to migrate to Chronicle Vesuvius.

I'd like @alexvotofuture and @awisniew207's opinion since I can't decide